scene/gui/tree.cpp

```diff
@@ -3606,6 +3612,7 @@ TreeItem *Tree::create_item(TreeItem *p_parent, int p_idx) {
 			ti = create_item(root, p_idx);
 		}
 	}
+	update();
```

Comment:
It's better to move this to TreeItem::create_child or tree items added with it won't update.

Reply:
Right, I somehow didn't notice this function.